scripts/test.sh

```
@@ -34,63 +52,220 @@ done
+    echo "❌ No output from ComfyUI API (curl command failed to produce output)"
+    echo "🔍 ComfyUI logs:"
+    docker exec "$CONTAINER_NAME" tail -15 /workspace/ComfyUI/user/comfyui.log 2>/dev/null || echo "No ComfyUI logs available"
+else
+    echo "✅ ComfyUI API is responding"
+    echo "📊 ComfyUI status:"
+    echo "$COMFYUI_QUEUE_OUTPUT" | head -5 2>/dev/null || echo "Could not parse queue status"
+fi
+    echo "❌ ComfyUI API is not responding (curl exit code: $CURL_EXIT_CODE)"
+    echo "🔍 ComfyUI logs:"
+    docker exec "$CONTAINER_NAME" tail -15 /workspace/ComfyUI/user/comfyui.log 2>/dev/null || echo "No ComfyUI logs available"
+fi
```

Shellcheck (0.11.0)

[error] Parsing stopped here. Is this keyword correctly matched up?

(SC1089)